src/components/Admin/AdminLocalisations.tsx

```diff
@@ -16,7 +16,11 @@ export const AdminLocalisations: React.FC = () => {
   const [exportDisabled, setExportDisabled] = useState(true)
   const [importDisabled, setImportDisabled] = useState(true)
   const showToast = useToast({ position: topLeft })
-  const [showModalWarning, setShowModalWarning] = useState<ModalProps>({ open: false })
+  const { ConfirmModal: WarningModal, showModal: showWarningModal } = useConfirmationModal({
+    type: 'warning',
+    title: strings.LOCALISATION_DELETE_WARNING_TITLE,
+    confirmText: strings.BUTTON_CONFIRM,
+  })
```

Comment:
All modal parameters can either be provided when the hook is first instantiated, or when the "showModal" method is called. As a general rule, I try to put them in the hook call except when either:
- the values might be dynamic (e.g. message dependent on context)
- they refer to items that haven't been defined yet (e.g. functions)